utils/build/virtual_machine/weblogs/dotnet/provision_test-app-dotnet-unsupported.yml

```diff
@@ -0,0 +1,45 @@
+lang_variant:
+    name: dotnet
+    cache: true
+    version: 3
+    install:
+      - os_type: linux
+        os_distro: deb
+        os_branch: ubuntu20_amd64
+        remote-command: |
+          sudo apt-get update;
+          sudo apt-get install -y apt-transport-https;
+          sudo apt-get update;
+          sudo apt-get install -y dotnet-sdk-2.1
+
+      - os_type: linux
```

Comment:
If you are going to run the test only for ubuntu 20 amd, you can remove this installation, because you have a specific installation for ubuntu 20 amd